src/main/kotlin/no/nav/syfo/unntak/api/UnntakApi.kt

```diff
@@ -55,8 +57,26 @@ fun Route.registerUnntakApi(
                 call.respond(unntakDTOList)
             }
         }
-        get(unntakApiHackaton) {
-            call.respond(unntakService.getHackaton(call.getNAVIdent()))
+        get(unntakApiStatistikk) {
+            val unntakForventetFriskmelding = unntakService.getUnntakForVeileder(call.getNAVIdent()).filter { unntak ->
+                unntak.arsak == UnntakArsak.FORVENTET_FRISKMELDING_INNEN_28UKER
+            }
+            val unntakStatistikkDTOList = unntakForventetFriskmelding.mapNotNull {
```

Comment:
Hvorfor flyttet du logikken inn hit? Jeg liker det best når api-filen er litt, og service-filene inneholder logikk 🤔

Reply:
Tenker at UnntakService ikke bør ha avhengighet til API-DTO-er, men bare Unntak eller andre domene-konsepter. Og så kan man bruke OppfolgingstilfelleService direkte for å finne tilfeller, i stedet for å gå via DialomotekandidatService. Man for så vidt enig i at det er fint med tynne API-filer... kanskje vi bør ha en UnntakStatistikkService eller en domeneklasse for UnntakStatistikk 🤔

Reply:
Enig i at DTO-objekt bare burde være i APIet, ja 👍 Hvis du får til noe enkelt som kan flytte ut litt kode, er jo det fint, ellers er det greit som det er 👍

Reply:
👍 Ser om jeg kan forbedre noe i ny PR (med cache - se over)